### ISSUE TITLE
Add cross validation to NeuralNet

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/deeprank-core#189]

**Comment by DaniBodor** [comment]
make sure output metrics are unified and not separate graphs for each cross validation

**Comment by DaniBodor** [comment]
Are we implementing this in deeprank-core or in 3DVac?

**Comment by gcroci2** [comment]
> Are we implementing this in deeprank-core or in 3DVac?

Yes so first it will be implemented there, then we can implement it here. I'll remove priority label since it's not priority for the generalization plan
